Tools/Pipeline/Common/PipelineController.FileWatcher.cs

```diff
@@ -0,0 +1,67 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System.IO;
+using System.Threading;
+
+namespace MonoGame.Tools.Pipeline
+{
+    internal partial class PipelineController
+    {
+        private class FileWatcher
```

Comment:
@cra0zy can you implement IDisposable and call Stop in the Dispose please. Just to make sure we shut this thread down